src/app/resources/resources.component.ts

```diff
@@ -66,6 +70,7 @@ export class ResourcesComponent implements OnInit, AfterViewInit {
 
   ngOnInit() {
     this.getResources();
+
```

Comment:
We can remove extra line here .

Reply:
Extra line removed, Please verify.
@nirojdyola 
Thanks for pointing that out.